tests/handlers.tests

```diff
@@ -1,490 +1,491 @@
 Identity handler (1)
 { handle(42) {case Return(x) -> x} }
 stdout : 42 : Int
-flags : --enable-handlers -e
```

Comment:
Why is the `-e` parameter removed from all these tests? I understand that it is now added automatically if `filemode : true`, but filemode does not seem to be enabled here.

Reply:
It's the other way around: The script adds `-e` as the last argument iff  `filemode : true` is *not* given.